src/contexts/useStockNews.js

```diff
@@ -5,35 +5,64 @@ const StockNewsContext = createContext({})
 
 const StockNewsProvider = ({children}) => {
     const [stockNews, setStockNews] = useState([]);
-    const [allNewsSources, setAllNewsSources] = useState([]);
-    const [allNewsInfo, setAllNewsInfo] = useState([]);
+    const [newsSelections, setNewsSelections] = useState([]);
+    const [newsMasterlist, setNewsMasterlist] = useState([]);
 
     const setNews = (stockSymbol) => {
         axios.get(`http://localhost:3000/stock-news/news/${stockSymbol}`).then((response) => {
             setStockNews(response.data.news);
         })
     }
 
-    const setNewsSources = () => {
-        axios.get(`http://localhost:3000/stock-news/allNewsSources`).then((response) => {
-            setAllNewsSources(response.data);
+    const setNewsSelections2 = () => {
+        axios.get(`http://localhost:3000/stock-news/newsSelections`).then((response) => {
+            setNewsSelections(response.data);
         })
     }
 
-    const setNewsInfo = () => {
-        axios.get('http://localhost:3000/stock-news/allNewsInfo').then((response) => {
-            setAllNewsInfo(response.data);
+    const setNewsMasterlist2 = () => {
```

Comment:
Can we please use more descriptive name that appending a 2 at the end

Reply:
Thanks for catching this, 100% agree with you. They were intended as placeholder names; I wanted to ask your opinions on naming conventions here